doc/api/querystring.md

```diff
@@ -4,81 +4,130 @@
+* `eq` {String}. The substring used to delimit keys and values in the
+  query string. Defaults to `'='`.
+* `options` {Object}
+  * `decodeURIComponent` {Function} The function to use when decoding
+    percent-encoded characters in the query string. Defaults to
+    `querystring.unescape()`.
+  * `maxKeys` {number} Specifies the maximum number of keys to parse.
+    Defaults to `1000`. Specify `0` to remove key counting limitations.
+
+The `querystring.parse()` method parses a URL query string into a map of key
+and value pairs.
 
-Options object may contain `maxKeys` property (equal to 1000 by default), it'll
-be used to limit processed keys. Set it to 0 to remove key count limitation.
+For example, the query string `'foo=bar&abc=xyz&abc=123'` is parsed into:
 
-Options object may contain `decodeURIComponent` property (`querystring.unescape` by default),
-it can be used to decode a `non-utf8` encoding string if necessary.
+```js
+{
+  foo: 'bar',
+  abc: ['xyz', '123']
+}
+```
 
-Example:
+*Note*: The object returned by the `querystring.parse()` method _does not_
+prototypically extend from the JavaScript `Object`. This means that the
+typical `Object` methods such as `obj.toString()`, `obj.hashOwnProperty()`,
+and others are not defined and *will not work*.
+
+By default, percent-encoded characters within the query string will be assumed
+to use UTF-8 encoding. If an alternative character encoding is used, then an
+alternative `decodeURIComponent` option will need to be specified as illustrated
+in the following example:
 
 ```js
-querystring.parse('foo=bar&baz=qux&baz=quux&corge')
-// returns { foo: 'bar', baz: ['qux', 'quux'], corge: '' }
+// Assuming gbkDecodeURIComponent function already exists...
 
-// Suppose gbkDecodeURIComponent function already exists,
-// it can decode `gbk` encoding string
 querystring.parse('w=%D6%D0%CE%C4&foo=bar', null, null,
   { decodeURIComponent: gbkDecodeURIComponent })
-// returns { w: '中文', foo: 'bar' }
 ```
 
 ## querystring.stringify(obj[, sep][, eq][, options])
 <!-- YAML
 added: v0.1.25
 -->
 
-Serialize an object to a query string.
-Optionally override the default separator (`'&'`) and assignment (`'='`)
-characters.
+* `obj` {Object} The object to serialize into a URL query string
+* `sep` {String} The substring used to delimit key and value pairs in the
+  query string. Defaults to `'&'`.
+* `eq` {String}. The substring used to delimit keys and values in the
+  query string. Defaults to `'='`.
+* `options`
+  * `encodeURIComponent` {Function} The function is use when encoding
+    percent-encoded characters in the query string. Defaults to
+    `querystring.escape()`.
 
-Options object may contain `encodeURIComponent` property (`querystring.escape` by default),
-it can be used to encode string with `non-utf8` encoding if necessary.
+The `querystring.stringify()` method produces a URL query string from a
+given `object` by iterating through the objects "own properties".
```

Comment:
1. Either drop the backticks (preferrable since it is consistent with the `parse()` description) or use the actual parameter name (`obj`).
2. s/objects/object's/